### PR TITLE
AkTuningTable proposal

### PR DESCRIPTION
**Pull request by marcussatellite**
Hi Aure,

New in this pull request:
Modified subclasses of AKPolyphonicNode to use AKTuningTable as default noteOn implementation.
Frequency from AKTuningTable is passed to C++/Soundpipe as a new noteOn function with frequency parameter.
Ideally should have no impact to existing users of AudioKit.
Added Scala file support to AKTuningTable.
I started adding a Microtonal playground page under Synthesis.

### CODE REVIEW

File: AudioKit/Common/Playgrounds/Synthesis.playground/contents.xcplayground [hunk 1]
```diff
@@ -1,36 +1,2 @@
...
-        <page name='PWM Oscillator'/>
-        <page name='PWM Oscillator Bank'/>
-        <page name='Sawtooth Wave Oscillator Operation'/>
-        <page name='Segment Operations'/>
-        <page name='Sporth Based Generator'/>
-        <page name='Telephone'/>
-    </pages>
-</playground>
\ No newline at end of file
+<playground version='6.0' target-platform='ios' display-mode='rendered'/>
```
Comment (aure) [inline AudioKit/Common/Playgrounds/Synthesis.playground/contents.xcplayground]: Can you revert this change?  Good to have the pages in there and it looks like your Xcode just accidentally bashed this.

Comment (marcussatellite) [inline AudioKit/Common/Playgrounds/Synthesis.playground/contents.xcplayground]: For sure...Trying to figure out how to revert this change only.

**Comment by aure** [comment]
I manually merged this
